### DISCUSSION TITLE
bad links

### DISCUSSION THREAD

**Discussion by bbannon** [craigk5n/webcalendar#359] [General]
Craig,
I'm getting 404 - page not found
for http://www.k5n.us/cknudsen
Did you forget to renew your domain?
Bruce

**Comment by craigk5n** [comment]
Reorganized it a while ago (Wordpress now).  Tried to redirect most links but missed that one.
The main webcalendar URL is: https://www.k5n.us/webcalendar

**Comment by bbannon** [comment]
OK. I've updated all those links I can find.

I was going through checking links.
I found several broken links to other places, but can't find the new URLs.
If any.
I'm working on marking them as [bad link].

Also found a few typos.

I'll commit all of these in the PR where I changed the URL to the license.
If you merge that in before I get the rest done, or not, it won't matter
either way.

Bruce
